src/translation/request.ts

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+const BASE_URL =
+  "https://translate.googleapis.com/translate_a/single?client=gtx&sl=";
+
+export async function createURL(source: string, target: string, text: string) {
```

Comment:
does this function create URL? I expected it would translate...

Reply:
renamed